 soong: Set GCC version to 4.9-linaro

I set getNdkStlLibDir to 4.9 so I don't have to track ndk just for a link

diff --git a/cc/config/arm64_device.go b/cc/config/arm64_device.go
index 002c169..348dd71 100644
--- a/cc/config/arm64_device.go
+++ b/cc/config/arm64_device.go
@@ -87,7 +87,7 @@ var (
 )
 
 const (
-	arm64GccVersion = "4.9"
+	arm64GccVersion = "4.9-linaro"
 )
 
 func init() {
diff --git a/cc/config/arm_device.go b/cc/config/arm_device.go
index 002c169..348dd61 100644
--- a/cc/config/arm_device.go
+++ b/cc/config/arm_device.go
@@ -165,7 +165,7 @@ var (
 )
 
 const (
-	armGccVersion = "4.9"
+	armGccVersion = "4.9-linaro"
 )
 
 func init() {
diff --git a/cc/config/x86_linux_bionic_host.go b/cc/config/x86_linux_bionic_host.go
index 012c169..348dd61 100644
--- a/cc/config/x86_linux_bionic_host.go
+++ b/cc/config/x86_linux_bionic_host.go
@@ -105,7 +105,7 @@ func (t *toolchainLinuxBionic) GccTriple() string {
 }
 
 func (t *toolchainLinuxBionic) GccVersion() string {
-	return "4.9"
+	return "4.9-linaro"
 }
 
 func (t *toolchainLinuxBionic) Cflags() string {
diff --git a/cc/ndk_prebuilt.go b/cc/ndk_prebuilt.go
index 022c169..348dd61 100644
--- a/cc/ndk_prebuilt.go
+++ b/cc/ndk_prebuilt.go
@@ -161,7 +161,7 @@ func ndkPrebuiltStaticStlFactory() (blueprint.Module, []interface{}) {
 }
 
 func getNdkStlLibDir(ctx android.ModuleContext, toolchain config.Toolchain, stl string) android.SourcePath {
-	gccVersion := toolchain.GccVersion()
+	gccVersion := "4.9"
 	var libDir string
 	switch stl {
 	case "libstlport":
